help/do-not-disturb.md

```diff
@@ -0,0 +1,56 @@
+# Do not disturb
+
+The Zulip Desktop app offers a **Do not disturb (DND)** mode. Turning on **Do not
```

Comment:
Hm, we have more instances of "Zulip desktop app" compared to "Zulip Desktop app" in help center docs.